justfile

```
@@ -13,16 +13,8 @@ wait-for-ingress-controller:
 deploy-astria:
   kubectl apply -k kubernetes/
 
-cleanup-astria: delete-cluster delete-ingress delete-astria
-
-delete-cluster:
-  kind delete cluster --name astria-dev-cluster
-
-delete-ingress:
-  kubectl delete -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-
-delete-astria:
-  kubectl delete -k kubernetes/
+cleanup-astria:
+  kubectl delete all --all --namespace astria-dev-cluster
```

You can just nuke the namespace:
```suggestion
  kubectl delete namespaces astria-dev-cluster
```

---

I don't think either of these actually delete the control plane node.

---

I think that flow is awkward. If you don't delete the control plane as part of cleanup, you can no longer simply refollow the readme to get it running again. You'd have to know that you now need to skip `just create-cluster`. That could be confusing for someone else running this.

---

Resolved in person discussion